XenModel/Messages.resx

```diff
@@ -3253,6 +3284,9 @@ Please reconnect the host and try again.</value>
   <data name="CREATED_VMPP" xml:space="preserve">
     <value>VM protection policy '{0}' created.</value>
   </data>
+  <data name="CREATED_VMSS" xml:space="preserve">
+    <value>VM Snapshot Schedule '{0}' created.</value>

```

Comment:
I think we should always refer to just "Snapshot schedule" not "VM snapshot schedule"